systems-ui/src/pages/dashboard/Dashboard.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from "react";
+import SystemsOrchestrator from "../../common/components/systems/systems-orchestrator/SystemsOrchestrator";
+import SystemOverview from "../../common/components/systems/system-overview/SystemOverview";
+import styles from "./Dashboard.css";
+import Loader from "../../common/components/loader/Loader";
+import SystemsGrid from "../../common/components/systems/systems-grid/SystemsGrid";
+import SystemsStatusOverview from "./systems-status-overview/SystemsStatusOverview";
+import Filter from "../../common/components/filter/Filter";
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <SystemsOrchestrator polling>
+        {
+          ({
+             systems,
+             isSystemsLoading,
+             error,
+             lastUpdated,
+           }) =>
+          {
+            if(isSystemsLoading) {
+              return (
+                <div className={styles.loaderContainer}>
+                  <Loader/>
+                </div>
+              );
+            }
+
+            if(error) {
+              return (
+                <div>An error happened while accessing the systems api</div>
```

Comment:
How would you approach the problem of supporting multiple languages for the UI ?

Reply:
I would probably inject the translations with a high order component and use keys instead of strings and let this HOC component decide which languages translations to inject. Using this library https://github.com/Tiqa/redux-polyglot/blob/master/src/translate.js I would do something like t('systems.detail.header')

We could also inject the translation in compile time for a better performance but this would really depend on the application requirements